examples/softk.jl

```diff
@@ -0,0 +1,226 @@
+"""
```

Comment:
As we discussed, let's maybe have two examples: one with plotting and one without?